src/scripts/parsers.js

```diff
@@ -0,0 +1,30 @@
+"use strict"
+
+export const parseTx = signMessage => {
+  const { msgs, fee, memo } = JSON.parse(signMessage)
```

Comment:
As per my other comment, I would avoid `JSON.parse()` being called 3 times.

Reply:
I reduced the amounts of `JSON.parse`